src/style.css

```diff
@@ -34,13 +37,14 @@ div {
 }
 
 #list-container {
-  border: 2px solid #101010;
+  border: 2px dashed #6edcd9;
   width: 100%;
   font-size: 12px;
 }
 
 li {
   border-bottom: 1px dashed #6edcd9;
+  color: #0c0c0c;
```

Comment:
- [ ] optional - currently ur button looks like this 
![image](https://user-images.githubusercontent.com/60612329/163631458-9fb1c36a-c621-462c-b9ea-1eff2e0322aa.png)
but I would suggest that you increase the padding inside the button to reveal the text 👍🏾

Reply:
Thank you, I will fix that.